sunpy/instr/lyra.py

```diff
@@ -631,9 +631,9 @@ def split_series_using_lytaf(timearray, data, lytaf):
     el = len(lytaf)
 
     # make the input time array a list of datetime objects
-    datetime_array = []
+    time_array = []
     for tim in timearray:
-        datetime_array.append(parse_time(tim))
+        time_array.append(parse_time(tim))
```

Comment:
should we not cast to a `Time` at the end here, to convert from a list of times to a Time with an array in it.

Reply:
I did and it fails the tests!!! (Don't know why, have to investigate it :D)

Reply:
lol ok, well this is fine if that's what it needs to be.